tabular/src/autogluon/tabular/models/rf/rf_rapids_model.py

```diff
@@ -10,17 +10,20 @@
 
 # TODO: Improve memory safety
 # TODO: Respect time limit
-# TODO: Significantly less accurate than RFModel with same hyperparameters.
-#  Refer to https://github.com/rapidsai/cuml/issues/2518
+# TODO: Depending on max_depth parameter, RFRapidsModel is slower than RFModel. 
```

Comment:
Thanks for mentioning this detail. If someone from RAPIDS team has a recommendation at present for a max_depth default value that maintains good accuracy without suffering too heavily from this bug, we can update the default max_depth (Currently 99).